### PR TITLE
Fix outdated doc

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
This is an automated comment for commit bf3dc24471bfd54d192aafac15999d50d26b4a3c with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/54393/bf3dc24471bfd54d192aafac15999d50d26b4a3c/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: docs/en/engines/table-engines/mergetree-family/mergetree.md [hunk 1]
```diff
@@ -1248,8 +1247,6 @@ Limit parameters (mainly for internal usage):
 
 Other parameters:
 * `metadata_path` - Path on local FS to store metadata files for Blob Storage. Default value is `/var/lib/clickhouse/disks/<disk_name>/`.
-* `cache_enabled` - Allows to cache mark and index files on local FS. Default value is `true`.
```
Comment (devcrafter) [inline docs/en/engines/table-engines/mergetree-family/mergetree.md:1251]: It'd be better to delete it in a more consistent way
```
─ rg -aF 'cache_enabled' .
./docs/en/operations/storing-data.md
163:                <data_cache_enabled>1</data_cache_enabled>

./programs/diagnostics/testdata/configs/xml/config.xml
1212:                <cache_enabled>true</cache_enabled>

./docs/zh/engines/table-engines/mergetree-family/mergetree.md
748:            <cache_enabled>true</cache_enabled>
775:- `cache_enabled` - 是否允许缓存标记和索引文件。默认值为`true`。

./docs/en/engines/table-engines/mergetree-family/mergetree.md
1223:            <cache_enabled>true</cache_enabled>
1251:* `cache_enabled` - Allows to cache mark and index files on local FS. Default value is `true`.

./docs/ru/engines/table-engines/mergetree-family/mergetree.md
808:            <cache_enabled>true</cache_enabled>
835:-   `cache_enabled` — признак, разрешено ли хранение кэша засечек и файлов индекса в локальной файловой системе. Значение по умолчанию: `true`.

./programs/server/config.xml
451:                <cache_enabled>true</cache_enabled>
```
The same for `cache_path`

**Review by devcrafter — DISMISSED** [review]
with comment

**Review by devcrafter — APPROVED** [review]

**Comment by Felixoid** [comment]
We should not merge PRs with failing builds

**Pull request merged**